### DISCUSSION TITLE
Elsa Studio 3.0 - failed to connect from elsa studio when using balzor server docker

### DISCUSSION THREAD

**Discussion by nurhidayat-agung** [elsa-workflows/elsa-core#5612] [Q&A]
hello elsa,

why i get error connection refuse while run docker blazor server app from my docker ?
it seem my local docker elsa server refised when hit by may elsa studio docker

**Comment by glime-ai** [comment]
To assist you with the connection refused error you're encountering when running the Elsa Blazor Server app from Docker, I need to gather a bit more information. Could you please provide the following details:

1. The exact command you used to run the Elsa Server Docker container.
2. The exact command you used to run the Elsa Studio Docker container.
3. Any environment variables or configuration settings you may have adjusted or added.
4. The error message or log output you're seeing when the connection is refused.

This information will help me understand your setup better and provide you with a more accurate solution. Additionally, ensure that both the Elsa Server and Elsa Studio containers are running on different ports and that the Elsa Studio is correctly configured to communicate with the Elsa Server's API endpoint.

If you have already followed the documentation and are still facing issues, please provide the requested details so I can help you troubleshoot the problem further.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
